video_player: Could not find com.google.android.exoplayer:exoplayer-core:2.8.0

<!-- Thank you for using Flutter!

     If you are looking for support, please check out our documentation
     or consider asking a question on Stack Overflow:
      * https://flutter.io/
      * https://docs.flutter.io/
      * https://stackoverflow.com/questions/tagged/flutter?sort=frequent

     If you have found a bug or if our documentation doesn't have an answer
     to what you're looking for, then fill our the template below. Please read
     our guide to filing a bug first:  https://flutter.io/bug-reports/
-->

## Steps to Reproduce

<!-- Please tell us exactly how to reproduce the problem you are running into. -->

1. attempt to run the `flutter_gallery`: `flutter run --release`

## Logs

<!--
      Include the full logs of the commands you are running between the lines
      with the backticks below. If you are running any "flutter" commands,
      please include the output of running them with "--verbose"; for example,
      the output of running "flutter --verbose create foo".
-->

```
westy92@westy92-VirtualBox:~/Source/flutter/examples/flutter_gallery$ flutter run --release
Initializing gradle...                                       1.4s
Resolving dependencies...                                   18.5s
Launching lib/main.dart on Pixel in release mode...
Gradle task 'assembleRelease'...                                 

FAILURE: Build failed with an exception.

* What went wrong:
Could not resolve all files for configuration ':app:releaseCompileClasspath'.
> Could not find com.google.android.exoplayer:exoplayer-core:2.8.0.
  Searched in the following locations:
    - file:/home/westy92/Android/Sdk/extras/m2repository/com/google/android/exoplayer/exoplayer-core/2.8.0/exoplayer-core-2.8.0.pom
    - file:/home/westy92/Android/Sdk/extras/m2repository/com/google/android/exoplayer/exoplayer-core/2.8.0/exoplayer-core-2.8.0.jar
    - file:/home/westy92/Android/Sdk/extras/google/m2repository/com/google/android/exoplayer/exoplayer-core/2.8.0/exoplayer-core-2.8.0.pom
    - file:/home/westy92/Android/Sdk/extras/google/m2repository/com/google/android/exoplayer/exoplayer-core/2.8.0/exoplayer-core-2.8.0.jar
    - file:/home/westy92/Android/Sdk/extras/android/m2repository/com/google/android/exoplayer/exoplayer-core/2.8.0/exoplayer-core-2.8.0.pom
    - file:/home/westy92/Android/Sdk/extras/android/m2repository/com/google/android/exoplayer/exoplayer-core/2.8.0/exoplayer-core-2.8.0.jar
    - https://dl.google.com/dl/android/maven2/com/google/android/exoplayer/exoplayer-core/2.8.0/exoplayer-core-2.8.0.pom
    - https://dl.google.com/dl/android/maven2/com/google/android/exoplayer/exoplayer-core/2.8.0/exoplayer-core-2.8.0.jar
    - https://jcenter.bintray.com/com/google/android/exoplayer/exoplayer-core/2.8.0/exoplayer-core-2.8.0.pom
    - https://jcenter.bintray.com/com/google/android/exoplayer/exoplayer-core/2.8.0/exoplayer-core-2.8.0.jar
  Required by:
      project :app > com.google.android.exoplayer:exoplayer:2.8.0
      project :app > com.google.android.exoplayer:exoplayer:2.8.0 > com.google.android.exoplayer:exoplayer-hls:2.8.0
      project :app > com.google.android.exoplayer:exoplayer:2.8.0 > com.google.android.exoplayer:exoplayer-smoothstreaming:2.8.0
      project :app > com.google.android.exoplayer:exoplayer:2.8.0 > com.google.android.exoplayer:exoplayer-ui:2.8.0
> Could not find com.google.android.exoplayer:exoplayer-dash:2.8.0.
  Searched in the following locations:
    - file:/home/westy92/Android/Sdk/extras/m2repository/com/google/android/exoplayer/exoplayer-dash/2.8.0/exoplayer-dash-2.8.0.pom
    - file:/home/westy92/Android/Sdk/extras/m2repository/com/google/android/exoplayer/exoplayer-dash/2.8.0/exoplayer-dash-2.8.0.jar
    - file:/home/westy92/Android/Sdk/extras/google/m2repository/com/google/android/exoplayer/exoplayer-dash/2.8.0/exoplayer-dash-2.8.0.pom
    - file:/home/westy92/Android/Sdk/extras/google/m2repository/com/google/android/exoplayer/exoplayer-dash/2.8.0/exoplayer-dash-2.8.0.jar
    - file:/home/westy92/Android/Sdk/extras/android/m2repository/com/google/android/exoplayer/exoplayer-dash/2.8.0/exoplayer-dash-2.8.0.pom
    - file:/home/westy92/Android/Sdk/extras/android/m2repository/com/google/android/exoplayer/exoplayer-dash/2.8.0/exoplayer-dash-2.8.0.jar
    - https://dl.google.com/dl/android/maven2/com/google/android/exoplayer/exoplayer-dash/2.8.0/exoplayer-dash-2.8.0.pom
    - https://dl.google.com/dl/android/maven2/com/google/android/exoplayer/exoplayer-dash/2.8.0/exoplayer-dash-2.8.0.jar
    - https://jcenter.bintray.com/com/google/android/exoplayer/exoplayer-dash/2.8.0/exoplayer-dash-2.8.0.pom
    - https://jcenter.bintray.com/com/google/android/exoplayer/exoplayer-dash/2.8.0/exoplayer-dash-2.8.0.jar
  Required by:
      project :app > com.google.android.exoplayer:exoplayer:2.8.0

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 6s
 
Gradle task 'assembleRelease'... Done                        7.4s
Gradle task assembleRelease failed with exit code 1
```

<!-- If possible, paste the output of running `flutter doctor -v` here. -->

```
[✓] Flutter (Channel stable, v1.0.0, on Linux, locale en_US.UTF-8)
    • Flutter version 1.0.0 at /home/westy92/Source/flutter
    • Framework revision 5391447fae (10 days ago), 2018-11-29 19:41:26 -0800
    • Engine revision 7375a0f414
    • Dart version 2.1.0 (build 2.1.0-dev.9.4 f9ebf21297)

[✓] Android toolchain - develop for Android devices (Android SDK 28.0.3)
    • Android SDK at /home/westy92/Android/Sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-28, build-tools 28.0.3
    • Java binary at: /home/westy92/android-studio/jre/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1136-b06)
    • All Android licenses accepted.

[✓] Android Studio (version 3.2)
    • Android Studio at /home/westy92/android-studio
    • Flutter plugin version 31.1.1
    • Dart plugin version 181.5656
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1136-b06)

[✓] VS Code (version 1.28.0)
    • VS Code at /usr/share/code
    • Flutter extension version 2.21.1

[✓] Connected device (1 available)
    • Pixel • FA69H0305733 • android-arm64 • Android 9 (API 28)

• No issues found!
```

I could only get this to work by adding a custom repository in `android/build.gradle`:
```
        maven {
            url 'http://repo.spring.io/plugins-release/'
        }
```

I'm guessing you will need to release an update to the `video_player` plugin and then update `flutter_gallery` to use it.

--- a/examples/flutter_gallery/android/build.gradle
+++ b/examples/flutter_gallery/android/build.gradle
@@ -13,6 +13,9 @@ allprojects {
     repositories {
         google()
         jcenter()
+        maven {
+            url 'https://google.bintray.com/exoplayer/'
+        }
     }
 }
 

